build.sh

```diff
@@ -28,7 +28,7 @@ do
   cat build/Dockerfile.prefix > Dockerfile
   cat "flavour/${FLAVOUR}/Dockerfile.flavour" >> Dockerfile
   cat build/Dockerfile.suffix >> Dockerfile
-  docker build . -t "dodevops/cloudcontrol-${FLAVOUR}:${TAG}"
```

Comment:
This is rather a local fix, but it shouldn't be problematic for the CI as well, right?

Reply:
You raise a question on your own PR? 😄
Well, this should not be an issue for github actions, but it is for local development: Can you make it cleaner and select the platform depending on the given FLAVOUR?
Since you already designed it kind of dynamic (searching for FLAVOURS or picking a Dockerfile from the given FLAVOUR in) it maybe makes sense to provide the supported architectures in a little text-file in the flavour folders?
I doubt that can't be used for the github actions yaml, so maybe a smarter idea is to get the platforms out of the github actions yaml (maybe via yq?), so it hasn't to be defined twice...

Reply:
I have removed my change to the build script for now. The build script is *only* used for local builds, the CI does it in another way and my idea is to add a PR with a better build script later with proper argument handling.